torch/_higher_order_ops/utils.py

```diff
@@ -264,10 +264,12 @@ def maybe_clone(t):
 def prepare_fw_with_masks(fn):
     def fw_with_masks(*args):
         fw_out = fn(*args)
-        return fw_out, [
+        fw_out_flat, tree_spec = pytree.tree_flatten(fw_out)
```

Comment:
Do we need this change?

Reply:
Good catch, no we don't. This was an artifact from resolving earlier issues